docs/userguide/declarative_config.rst

```diff
@@ -259,3 +253,40 @@ order to include markers, they **must** be *dangling*:
       ReportLab>=1.2
       RXP
       importlib-metadata; python_version < "3.8"
+
+
+Compatibility with other tools
+==============================
+
+Historically, several tools explored declarative package configuration
+in parallel. And several of them chose to place the packaging
+configuration within the project's :file:`setup.cfg` file.
+One of the first was ``distutils2``, which development has stopped in
+2013. Other include ``pbr`` which is still under active development or
+``d2to1``, which was a plug-in that backports declarative configuration
+to ``distutils``, but has had no release since Oct. 2015.
+As a way to harmonize packaging tools, ``setuptools``, having held the
+position of *de facto* standard, has gradually integrated those
+features as part of its core features.
+
+Still this has lead to some confusion and feature incompatibilities:
+
+- some tools support features others don't;
+- some have similar features but the declarative syntax differs;
+
+The table below tries to summarize the differences. But, please, refer
+to each tool documentation for up-to-date information.
+
+=========================== ========== ========== ===== ===
+feature                     setuptools distutils2 d2to1 pbr
+=========================== ========== ========== ===== ===
+[metadata] description-file N          Y          Y     Y
```

Comment:
I think it's
```suggestion
[metadata] description-file S          Y          Y     Y
```
because in setuptools you can do `long_description = file: README.rst, CHANGELOG.rst` to achieve the same outcome.